VKApp/VKApp/TabBar/First/PhotosCollectionViewCell.swift

```diff
@@ -26,8 +30,8 @@ final class PhotosCollectionViewCell: UICollectionViewCell {
         userPhotoImageView.layer.add(sizeAnimate, forKey: nil)
     }
 
-    func configure(_ image: UIImage) {
-        userPhotoImageView.image = image
+    func configure(_ images: UIImage) {
```

Comment:
```suggestion
    func configure(_ image: UIImage) {
```
передаем же 1 элемент